go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/360EntSecGroup-Skylar/excelize/v2
```

Comment:
When you are developing locally with go modules you can use `replace` to change where the code is pulled from to test your local changes instead of renaming the module.